index.html

```diff
@@ -214,5 +214,70 @@ <h2>
         </div>
       </div>
     </div>
+    <div class="contact-section">
+      <p class="para-info">
+        I'm always <br />interested in hearing<br />
+        about new projects,<br />
+        so if you'd like to<br />
+        chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/{form_id}">
+        <ul class="list-container">
+          <li>
+            <input
+              type="text"
+              placeholder="Full name"
+              class="user-info"
+              maxlength="10"
+              required
```

Comment:
- [ ] [REQUIRED] Logically difficult for anyone to have a full name of `10` characters, please increase the value of maxlength property to `30` 🙏🏼